src/components/sections/TheNewArrivals.vue

```diff
@@ -0,0 +1,234 @@
+<template>
+  <section class="arrival spad">
+    <div class="container">
+      <div class="row">
+        <div class="col-lg-12 text-center">
+          <div class="section-title">
+            <span>Hoa tươi</span>
+            <h2>Mới Nhập Về</h2>
+          </div>
+          <div class="filter__controls">
+            <ul>
+              <li
+                v-for="item in filterSet"
+                :key="item.id"
+                :class="{ active: selectedFilterId == item.id }"
+                @click="selectedFilterId = item.id"
+              >
+                {{ item.filterName }}
+              </li>
+            </ul>
+          </div>
+        </div>
+      </div>
+      <div class="row">
+        <template v-for="item in newArrivalsItems">
+          <div
+            v-show="selectedFilterId == 0 || item.filterId == selectedFilterId"
+            :key="item.productId"
+            class="d-col"
+          >
+            <div class="product__item">
+              <div
+                :style="{
+                  'background-image': 'url(' + item.bgImgUrl + ')',
+                }"
+                class="product__item__pic set-bg"
+              >
+                <div v-if="item.label != ''" class="label">
```

Comment:
```suggestion
                <div v-if="item.label" class="label">
```
Như này cho ngắn gọn a ạ